main.go

```diff
@@ -225,6 +253,13 @@ func initFlags(fs *pflag.FlagSet) {
 		":9440",
 		"The address the health endpoint binds to.",
 	)
+	tlsCipherPreferredValues := cliflag.PreferredTLSCipherNames()
+	tlsCipherInsecureValues := cliflag.InsecureTLSCipherNames()
+	fs.StringVar(&tlsOptions.TLSCipherSuites, "tls-cipher-suites", "",
```

Comment:
No flags whatsoever should be added in this PR. We only want to disable TLS 1.0 and TLS 1.1. Cipher suites are staying as default like they were before.